### ISSUE TITLE
Validation Documentation Is Vague

### ISSUE THREAD

**Issue by drewcapener** [adam-cowley/neode#163]
Hi, I just wanted to drop in here and see if I could get some cooperation to update the documentation a bit. I understand the validation is done using Joi, and I have been able to triangulate between the Neode and Joi documentation to achieve the result I'm looking for for the most part, but certain things remain opaque to me. For example, I am having trouble understanding how conditional validation should work.

@adam-cowley I would be happy to help with this to the extent possible, but I would need help actually understanding how things work.

**Comment by joshamaju** [comment]
Did you get any response privately?

**Comment by drewcapener** [comment]
No, I stopped using Neode because of this and other challenges. Seemed like it wasn't being maintained...
